examples/reference/elements/bokeh/Spikes.ipynb

```diff
@@ -57,9 +58,9 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "%%opts Points (color='red')\n",
     "ys = np.random.rand(50)\n",
-    "hv.Points((xs, ys)) * hv.Spikes(xs)"
+    "overlay = hv.Points((xs, ys)) * hv.Spikes(xs)\n",
+    "overlay.opts(opts.Points(color='red'))"
```

Comment:
This would fall under the "single element styled inline:

```python
points = hv.Points((xs, ys))
spikes = hv.Spikes(xs)
points.opts(color='red') * spikes
```

but I think that can go either way.

Reply:
I think I prefer your suggestion accepting the new rule.